winlogbeat/eventlog/wineventlog.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/winlogbeat/sys"
+	"github.com/elastic/beats/winlogbeat/sys/wineventlog"
```

Comment:
For consistency, could you use the "aliased" import below for this package. It's just aliased to `win`.